lib/Fhp/Model/SEPAAccount.php

```diff
@@ -20,6 +20,14 @@ class SEPAAccount
     /** @var string|null */
     protected $blz;
 
+    /**
+     * Determines how single lines in transaction descriptions are joined, see #481.
+     * Defaults to an empty string for maximum compatibility but some banks implicitly assume line breaks for this.
```

Comment:
I'd say "backwards compatibility" (with previous versions of this library) rather than "maximum compatibility" (with the largest number of banks).